addon/components/layout/left-menu/component.js

```diff
@@ -6,7 +6,7 @@ import { camelize } from '@ember/string';
 
 import layout from './template';
 
-const CANNY_URL = 'https://upfluence.canny.io/feature-requests';
+const FEATURES_REQUEST_URL = 'https://portal.productboard.com/j1s1lqcvra9dybyfsrel1zyh';
 
 export default Component.extend({
```

Comment:
How about moving this component to Octane ? 🙂 
Otherwise, lgtm

Reply:
It's just a question of time but I can do it :slightly_smiling_face:

Reply:
Just the conversion of `observer` can be tricky